src/Domain/StripeWebhook.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace MatchBot\Domain;
```

Comment:
This is not part of our domain – it doesn't have a persisted database object or support one. I think `Application\Actions\Hooks\Stripe` or possible `Application\HttpModels` would be more suitable homes.